.github/workflows/main.yml

```diff
@@ -50,6 +50,34 @@ jobs:
           name: screenshots-${{ matrix.os }}-${{ matrix.version }}
           path: ${{ github.workspace }}/test/**/screenshots/*.png
 
+  publish-wiki:
+    runs-on: ubuntu-latest
+    name: Publish wiki
+    defaults:
+      run:
+        working-directory: ${{ github.workspace }}
+
+    env:
+      ORG: 'redhat-developer'
+      REPO_SOURCE: 'vscode-extension-tester'
+
+    steps:
+      - name: Checkout main repository
+        uses: actions/checkout@v4
+        with:
+          repository: ${{ format('{0}/{1}', env.ORG, env.REPO_SOURCE) }}
+          path: ${{ format('{0}/{1}', github.workspace, env.REPO_SOURCE) }}
+
+      - name: Upload Release Notes to Wiki
+        uses: docker://decathlon/wiki-page-creator-action:latest
+        env:
+          ACTION_MAIL: action@github.com
+          ACTION_NAME: GitHub Action
+          GH_PAT: ${{ secrets.GH_PAT }}
```

Comment:
from where is this secret taken?

Reply:
I was under the impression that this is a builtin secret, but I mightve confused it with something else.
I think `GITHUB_TOKEN` should do the trick, I'll double check that.

Reply:
yes, because I would expect there needs to be some token on side of this project to allow this bot write into project wiki pages